sql/core/src/test/scala/org/apache/spark/sql/streaming/FlatMapGroupsInPandasWithStateSuite.scala

```diff
@@ -738,4 +738,89 @@ class FlatMapGroupsInPandasWithStateSuite extends StateStoreMetricsTest {
       }
     }
   }
+
+  test("SPARK-40670: applyInPandasWithState - streaming having non-null columns") {
+    // scalastyle:off assume
```

Comment:
Oh, let's remove this. We don't have this rule in Apache Spark.

Reply:
lol My bad.

Reply:
Looks like we have all the "style off and on" in all test cases. Maybe a separate PR?

Reply:
sure!